spec/integration/urn_date_dim_spec.rb

```diff
@@ -0,0 +1,53 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata'
+
+describe "Blueprint now support urn in date dimension", :constraint => 'slow' do
+  before(:all) do
+    @client = ConnectionHelper.create_default_connection
+
+    @blueprint = GoodData::Model::ProjectBlueprint.build("My project from blueprint") do |p|
+      p.add_date_dimension('created_on', urn: 'urn:pe:date')
+    end
+  end
+
+  after(:all) do
+    @client && @client.disconnect
+  end
+
+  context 'project creates from blueprint contains urn in date dimension' do
+    before do
```

Comment:
Please use `before :all` and `after :all` so that we don't recreate the project every time.